templates/repo/diff/box.tmpl

```diff
@@ -79,7 +79,7 @@
 					</h4>
 				</div>
 			{{else}}
-				<div class="diff-file-box diff-box file-content {{TabSizeClass $.Editorconfig $file.Name}}" id="diff-{{.Index}}">
+				<div class="diff-file-box diff-box file-content {{if $.Editorconfig}}{{TabSizeClass $.Editorconfig $file.Name}}{{else}}tab-size-8{{end}}" id="diff-{{.Index}}">
```

Comment:
```suggestion
				<div class="diff-file-box diff-box file-content {{if $.Editorconfig}}{{TabSizeClass $.Editorconfig $file.Name}}{{else}}tab-size-4{{end}}" id="diff-{{.Index}}">
```

I think 4 is a better default. 8 is rather archaic.

Reply:
Hmm, It is kept same with ``TabSizeClass`` define now. Maybe your change is meaningful, but I don't like to change it in this pr, because it's not the main aim of this pr. Thanks. 

ref:
https://github.com/go-gitea/gitea/blob/e3b719863b7646ca171be0ecf5b6c52ef4a1a3a0/modules/templates/helper.go#L177-L178

Reply:
I see, yeah keep it at 8 for now.